src/main/resources/api/service_openapi.yaml

```diff
@@ -1969,6 +2012,21 @@ components:
             The tag of the container image. If not specified, this defaults to the latest tag.
           type: string
 
+    DeleteControlledGcpAiNotebookInstanceRequest:
+      type: object
+      required: [ jobControl ]
+      properties:
+        jobControl:
+          $ref: '#/components/schemas/JobControl'
+
+    DeleteControlledGcpAiNotebookInstanceResult:
+      type: object
+      properties:
+        jobReport:
```

Comment:
This is a little confusing, having both report objects in this "happy" result object. If you only return it for 200 or 202, why do we need an `ErrorReport`?

Reply:
You can have an ErrorReport in a happy result object is that the contained ErrorReport describes how the job failed, if present. The 20x happy response is that the client successfully polled the result endpoint and got a response from the service. Whether or not the job they're polling about finished successfully or had an error is encoded in the `jobReport` with more details in the `errorReport`. There's a difference between whether result endpoint worked and whether the job worked.

I'm following the [Async REST API spec](https://docs.google.com/document/d/1PTd4xvmV9xnEkWaIgFc6d3VUyJwwymQTFdhpKrdsUKw/edit), and the other APIs already implemented in WSM.  We can consider if there's another way we want to document its patterns for clients to read about.